docs/release.md

```diff
@@ -98,6 +98,31 @@ The last step of the release process creates artifacts for the new version of th
 
 1. `unknown revision internal/coreinternal/v0.55.0` -- This is typically an indication that there's a dependency on a new module. You can fix it by adding a new `replaces` entry to the `go.mod` for the affected module. 
 
+## Bugfix releases
+
+### Bugfix release criteria
+
+Both `opentelemetry-collector` and `opentelemetry-collector-contrib` have very short 2 week release cycles. Because of this, we put a high bar when considering making a patch release, to avoid wasting engineering time unnecessarily.
+
+When considering making a bugfix release on the `v0.N.x` release cycle, the bug in question needs to fulfill the following criteria:
+
+1. The bug was introduced on the `v0.N-1.x` release cycle.
```

Comment:
Does this mean in order to release a `0.55.1` patch, the bug had to appear in `0.55-1.0`, which is `0.54.0`?  Am I interpreting this correctly?

Reply:
Yes, that's it. Do you think I should phrase it differently?

Reply:
I think I was expecting a patch release to deal with issues that appeared in the release itself.  Like if 0.55.0 was released in a state such that the collector wouldn't even start, I was expecting a 0.55.1 release immediately, not waiting for 0.56.0 and then 0.56.1.

Combined with criteria #2, it sounds like in order for an issue from 0.55.0 to get patched, it has to be found on Wednesday/Thursday, Friday, or Saturday after the 0.55.0 release, then wait 11 more days for a 0.56.0 release, and then wait for a 0.56.1 release.

Should the first criteria be "The bug was introduced in version `v0.N-1.x` or higher, up to and including the current release"?

Reply:
Right, sorry, I meant `v0.N.0`, I don't know why I wrote that :smile:

> I was expecting a patch release to deal with issues that appeared in the release itself.

This is what I wanted to write, let me rephrase that

Reply:
Replaced `N-1`by `N`, hopefully that's clear enough